code/game/gamemodes/sandbox/h_sandbox.dm

```diff
@@ -29,7 +29,7 @@ GLOBAL_VAR_INIT(hsboxspawn, TRUE)
 	//items that shouldn't spawn on the floor because they would bug or act weird
 	var/static/list/spawn_forbidden = list(
 		/obj/item/tk_grab, /obj/item/implant, // not implanter, the actual thing that is inside you
-		/obj/item/assembly, /obj/item/onetankbomb,
+		/obj/item/assembly, /obj/item/pda/ai,
```

Comment:
this doesn't exist and also looks like a mistake

Reply:
Merge conflicts were on my ass about it, will just remove

Reply:
Oh, I see what this is now
Probably include ai cameras and modular interfaces (tablet/integrated) in this